README.md

```diff
@@ -24,37 +22,21 @@ providers:
-  --worker-machine-count=3 \
-  > sample.yaml
-```
-
-Virtink cluster OS environment variables.
-
-| Variable name                          | Note                                                       |
-|----------------------------------------|------------------------------------------------------------|
-| VIRTINK_POD_NETWORK_CIDR               | Virtink workload Kubernetes cluster Pod network CIDR       |
-| VIRTINK_SERVICE_CIDR                   | Virtink workload Kubernetes cluster Service network CIDR   |
-| VIRTINK_MACHINE_CPU_COUNT              | Virtink VM VCPU count                                      |
-| VIRTINK_MACHINE_MEMORY_SIZE            | Virtink VM memory size                                     |
-| VIRTINK_MACHINE_KERNEL_IMAGE           | Virtink VM kernel image                                    |
-| VIRTINK_MACHINE_ROOTFS_IMAGE           | Virtink VM rootfs image                                    |
-| VIRTINK_MACHINE_ROOTFS_SIZE            | Virtink VM rootfs size                                     |
-| VIRTINK_INFRA_CLUSTER_SECRET_NAME      | Virtink Infrastructure cluster kubeconfig Secret name      |
-| VIRTINK_INFRA_CLUSTER_SECRET_NAMESPACE | Virtink Infrastructure cluster kubeconfig Secret namespace |
-
-Cluster API common variables, see [Common variables](https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#common-variables).
+## Environment variables
+
+Except for the [common variables](https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#common-variables) provided by Cluster API, you can further customize your workload cluster on Virtink with following environment variables:
+
+| Variable name                          | Note                                                                                                                 |
+| -------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| VIRTINK_INFRA_CLUSTER_SECRET_NAME      | The name of secret in the management cluster that contains the kubeconfig of the Virtink infrastructure cluster      |
+| VIRTINK_INFRA_CLUSTER_SECRET_NAMESPACE | The namespace of secret in the management cluster that contains the kubeconfig of the Virtink infrastructure cluster |
+| VIRTINK_POD_NETWORK_CIDR               | The pod network CIDR to use for the workload cluster (default `10.17.0.0/16`)                                        |
+| VIRTINK_SERVICE_CIDR                   | The service network CIDR to use for the workload cluster (default `10.112.0.0/12`)                                   |
+| VIRTINK_MACHINE_CPU_COUNT              | The number of VM vCPUs (default `2`)                                                                                 |
+| VIRTINK_MACHINE_MEMORY_SIZE            | The memory size of VM (default `2GiB`)                                                                               |
```

Comment:
As prompted by default value, It's likely to set the memory size unit to GiB, which is invalid.
Not sure [drone/envsubst](https://github.com/drone/envsubst) can replace 'GiB' to 'Gi' and use a default value in the same time?